tests/tests_suite_test.go

```diff
@@ -32,7 +34,8 @@ import (
 )
 
 func TestTests(t *testing.T) {
-	reporters := []Reporter{reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"))}
+	maxFails := getMaxFailsFromEnv()
```

Comment:
Can you move this part before you define the flag and use the outcome as the "default" value? Let's follow the usual order:

 1) no env, no arg -> use the default
 2) env, no arg  -> use the env
 3) env, arg -> use the arg (so explicit arg beats env)

Reply:
Well the thing is, the reporter in the tests is consuming only the env variable while flag is only in the dump executable (which in turn do not read the env variable).

I did that because it felt more natural to be a parameter in the command without having to duplicate all the env parsing logic. If you think that this is confusing and both the tests / the dump command should have the flag AND the env variable I'll change them.

Reply:
I would say, if we have both, keep the expected order. If we only have envs, then I am fine :)

Reply:
Yes that's the case.
When running the test executable we only have env, when running the dump command we only have the flag.

Reply:
Perfect. Thanks for pointing me to it.